wire-ios-sync-engine/Source/Use cases/ResolveOneOnOneConversationsUseCase.swift

```diff
@@ -25,20 +27,22 @@ public protocol ResolveOneOnOneConversationsUseCaseProtocol {
 
 }
 
+typealias PullSelfUserClientsFactory = (NSManagedObjectContext) -> PullSelfUserClientsProtocol
```

Comment:
we use a closure to provide the PullSelfUserClients object because it depends on apiService which won't be ready when creating the strategies